File: Dermatoloska_ordinacija/UI/dermatoloska_desktop/lib/models/termin.dart
import 'package:json_annotation/json_annotation.dart';

part 'termin.g.dart';

@JsonSerializable()
class Termin{
	int? terminId;
  int? korisnikIdDoktor; 
  int? korisnikIdPacijent; 
  DateTime? datum;
  Termin(this.terminId, this.korisnikIdDoktor, this.korisnikIdPacijent, this.datum);

  factory Termin.fromJson(Map<String, dynamic> json) => _$TerminFromJson(json);

  Map<String, dynamic> toJson() => _$TerminToJson(this);

}

File: DermOrdDodatni/DermOrdDodatni/Database/Termin.cs
using System;
using System.Collections.Generic;

namespace DermOrdDodatni.Database;


public partial class Termin
{
    public int TerminId { get; set; }

    public DateTime? Datum { get; set; }

    public int? KorisnikIdDoktor { get; set; }

    public int? KorisnikIdPacijent { get; set; }

    public virtual Korisnik? KorisnikIdDoktorNavigation { get; set; }

    public virtual Korisnik? KorisnikIdPacijentNavigation { get; set; }
}
